_summaries/reddi2019on_1.md

```diff
@@ -1,19 +1,53 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: reddi2019on
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: aabayomi # Your GitHub id
+score: 8/10 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
+<!-- TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Summary 1: ON THE CONVERGENCE OF ADAM AND BEYOND
+
+## What is the core idea? ##
+
+The problem is this paper is trying to solve is the limitation of optimizers such as ADAM in a convex setting. However the solution to this is to have history of past gradients.
+
+### How is it realized (technically)? ###
+
+First is to prove the non-convergence of ADAM.
```

Comment:
could skip the proof, but again, I think should talk about why it happens with high-level descriptions